poe-tasks/lib/parse_args.sh

```diff
@@ -26,11 +26,11 @@ while [[ $# -gt 0 ]]; do
       shift
       ;;
     --name=*)
-      connectors=("${1#*=}")
+      connectors+=("${1#*=}")
       shift
       ;;
     --name)
-      connectors=("$2")
+      connectors+=("$2")
```

Comment:
this change from `=` -> `+=` is to match airbyte-ci behavior (i.e being able to pass multiple `--name source-foo` flags)